src/hooks/useRovingTabIndex.ts

```diff
@@ -12,6 +12,15 @@ export function useRovingTabIndex(isSelected: boolean) {
   function onFocus(event: React.FocusEvent<HTMLDivElement>) {
```

Comment:
When a cell is clicked it is automatically focused as the cell has a `tabIndex: -1`. We can focus the child here